content/rs/release-notes/rs-7-2-releases/rs-7-2-tba.md

```diff
@@ -28,11 +48,293 @@ The following table shows the MD5 checksums for the available packages:
+ 
+    1. Upgrade all databases to version 7.x or later.
+
+1. Enable RESP3 support for your database (`enabled` by default):
+
+    ```sh
+    rladmin tune db db:<ID> resp3 enabled
+    ```
+
+If you run Redis Stack commands with Redis clients [Go-Redis](https://redis.uptrace.dev/) version 9 or [Lettuce](https://lettuce.io/) versions 6 and later, see [client prerequisites](#client-prerequisites-for-redis-72-upgrade) before you upgrade to Redis 7.2 to learn how to prevent potential application issues due to RESP3 breaking changes.
+
+#### Sharded pub/sub
+
+[Sharded pub/sub](https://redis.io/docs/interact/pubsub/#sharded-pubsub) is now supported.
+
+You cannot use sharded pub/sub if you [deactivate RESP3 support]({{<relref "/rs/references/compatibility/resp#deactivate-resp3">}}).
+
+#### New Cluster Manager UI preview
+
+A preview of the redesigned Cluster Manager UI (admin console) is available in Redis Enterprise Software version 7.2.
+
+To try out the new UI:
+
+- On the sign-in screen:
+
+    1. Enter your credentials.
+
+    1. Select **Sign in the new interface**.
+
+- Sign in directly from the new UI's sign-in screen at <nobr>`https://<hostname or IP address>:8443/new`</nobr>
+
+- If you are currently signed in to the legacy UI:
+
+    1. Select **Switch to the new Admin Console** to expand the banner at the top of the screen.
+    
+    1. Click the **Try it now** button to open the new UI in another tab.
+
+##### New UI benefits
+
+- User-driven design
+
+- Provides full functionality to complete tasks entirely in the UI
+
+- New attributes and improved feature visibility
+
+- Provides configuration flexibility while highlighting the recommended path
+
+- Addresses the needs of different personas and use cases
+
+- Quicker troubleshooting and easier maintenance
+
+##### New UI highlights
+
+- More configurable database attributes, including replica high availability, shards placement, and proxy policy.
+
+- Nodes indicate whether it’s a primary or secondary node.
+
+- Modules show the databases that are using them.
+
+- Certificates show expiration and validity, and you can upload and copy certificates.
+
+- Cluster license displays the number of shards that are used out of the number of shards that are licensed to the cluster. The new UI allows you to paste or upload a new license.
+
+- Role-based access control (RBAC) has explanations to improve clarity. 
+
+- Access Control List (ACLs) now support defining ACLs for modules.
+
+- The databases screen has more information per database for faster troubleshooting. It also allows you to filter databases and compare database metrics.
+
+- The cluster name, user, and user role are shown in the upper right for quickly identifying the cluster from any screen. You can also **Change user password** from the dropdown menu.
+
+- Auto Tiering licensing and an emergency switch for the flash engine (available only in the new UI).
```

Comment:
I don't like this "emergency switch". It makes it feel like we are not confident in the new feature. The UI and other documentation pieces have enough warnings to reflect when to use and how.
```suggestion
- Auto Tiering licensing and a toggle for the storage engine used in Auto Tiering enabled databases (available only in the new UI).
```